lighthouse-core/formatters/critical-request-chains.js

```diff
@@ -240,17 +275,30 @@ class CriticalRequestChains extends Formatter {
           newTreeMarkers.push(!parentIsLastChild);
         }
 
-        if (!startTime) {
-          startTime = node.request.startTime;
-        }
-
         return opts.fn({
           node,
           isLastChild,
           hasChildren,
           startTime,
+          transferSize: (transferSize + node.request.transferSize),
           treeMarkers: newTreeMarkers
         });
+      },
+
+      createGlobalContext(tree, opts) {
```

Comment:
not totally clear what this global context is? sounds like a javascript global context.